tests/subsys/bluetooth/mesh/sensor_subsys/src/main.c

```diff
@@ -1234,6 +1234,93 @@ static void apparent_power_check(const struct bt_mesh_sensor_type *sensor_type)
 	}
 }
 
+static void temp_range_check(const struct bt_mesh_sensor_type *sensor_type)
+{
+	struct sensor_value in_value[sensor_type->channel_count];
+	int32_t test_vector[] = {0x8000, -273, 0, 100, 327};
```

Comment:
Probably `INT16_MIN` to understand better the purpose

Reply:
Type should be int32_t as this is used as val1 in a sensor_value which is 32 bit. It would work with int16_t but I don't see the enormous value, users of this API that I'm testing here will be using `int32_t`s. But I'll put INT16_MIN in there 👍

Reply:
Right, so INT16_MIN will put a negative int32_t here, while the sensor type is doing a positive int32_t = 0x8000. I think doing 0x8000 explicitly is fine here, as this is the literal hex value mentioned in the spec.